src/lucky_flow.cr

```diff
@@ -16,8 +16,13 @@ class LuckyFlow
     setting base_uri : String
     setting retry_delay : Time::Span = 10.milliseconds
     setting stop_retrying_after : Time::Span = 1.second
-    setting chromedriver_path : String?
+    setting driver_path : String?
     setting browser_binary : String? = nil
+    setting driver : LuckyFlow::Driver.class = LuckyFlow::Drivers::HeadlessChrome
+  end
+
+  def HabitatSettings.chromedriver_path=(_chromedriver_path)
```

Comment:
slightly weird way to define this method but it's the only way I found to do it.

Reply:
oh right! My bad. I was thinking just a regular method, but I guess you'd only call it from `settings.chromedriver_path`, but that's still awesome you could do it 😂